experiment/compatibility-versions/compatibility-versions-feature-gate-test.sh

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
```

Comment:
this can be a follow-up but we usually use `#!/usr/bin/env bash`, this guarantees that we use the preferred copy in PATH and is a little more portable (`/usr/bin/env` is virtually guaranteed due to `env` being POSIX and so many scripts doing this, but `bash` may not be in `/bin`, e.g. on my work mac I have installed newer bash via homebrew and kubernetes scripts may require that)

same goes for the other script.

https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability